components/network/quagga/patches/40-cve-2018.patch

```diff
@@ -0,0 +1,385 @@
+This patch includes the fixes for the following CVEs:
+cve-2018-5378
+cve-2018-5379
+cve-2018-5381
+cve-2017-16227
+cve-2016-1245
+
+All of these CVEs are fixed in Quagga 1.2.4 and this patch may be removed
+when Quagga in Solaris is upgraded to that version or later.
+
+These bug fixes are all directly from the Quagga community GIT repository at
+http://git.savannah.gnu.org/gitweb/?p=quagga.git. The specific changes are as
+follows:
+cve-2018-5378
+http://git.savannah.gnu.org/gitweb/?p=quagga.git;a=commitdiff;h=9e5251151894aefdf8e9392a2371615222119ad8
+
+cve-2018-5379
+http://git.savannah.gnu.org/gitweb/?p=quagga.git;a=commitdiff;h=e69b535f92eafb599329bf725d9b4c6fd5d7fded
+
+cve-2018-5381
+http://git.savannah.gnu.org/gitweb/?p=quagga.git;a=commitdiff;h=ce07207c50a3d1f05d6dd49b5294282e59749787
+
+cve-2017-16227
+http://git.savannah.gnu.org/gitweb/?p=quagga.git;a=commitdiff;h=7a42b78be9a4108d98833069a88e6fddb9285008
+
+cve-2016-1245
+http://git.savannah.gnu.org/gitweb/?p=quagga.git;a=commitdiff;h=23ed2c2fb49b8a15ad125b16278e535719d64e7d
+
+*** zebra/rtadv.c
```

Comment:
Can we convert patch to unified format?

Reply:
sorry, again if we can, I'd like to stick to the solaris-userland as close as we can (that might make it easier to update at some point when Oracle did the work)

Reply:
Converting patches is not so hard, but unified format is much easier to read and is a common practice